tests/cli.test.ts

```diff
@@ -0,0 +1,33 @@
+const { spawn } = require('child_process');
+
+import { Server } from './../src/server';
+import { Utils } from './utils';
+
+jest.retryTimes(parseInt(process.env.RETRY_TIMES || '1'));
+
+describe('CLI test', () => {
+    beforeEach(() => {
+        jest.resetModules();
+    });
+
+    test('soketi-pm2 reads config parameter', done => {
+        const pm2 = spawn('bin/pm2.js', ['start', '--config=tests/fixtures/config.json']);
+        setTimeout(() => {
+            throw new Error('Config file wasn\'t loaded.');
```

Comment:
If pm2.js didn't output `127.0.0.1:13337` (proving the config file was read) in 5 seconds, we consider the test failed